terratorch/datamodules/generic_multimodal_data_module.py

```diff
@@ -448,8 +460,26 @@ def __init__(
         self.predict_dataset_bands = predict_dataset_bands or dataset_bands
         self.predict_output_bands = predict_output_bands or output_bands
 
-        self.rgb_modality = rgb_modality or modalities[0]
-        self.rgb_indices = rgb_indices
+        # TODO: Remove in future release
```

Comment:
@blumenstiel can you please add more details why this has to be removed in future releases?